web/massastation/src/pages/Index/Index.tsx

```diff
@@ -1,20 +1,126 @@
+
+  const { data: availablePlugins } =
+    useResource<PluginStoreItemRequest[]>('plugin-store');
+
+  const { mutate } = usePost<any, any>(`plugin-manager?source=${urlPlugin}`);
+
+  useEffect(() => {
+    const isWalletInstalled = plugins?.some(
+      (plugin: PluginHomePage) => plugin.name === 'wallet',
+    );
+    setPluginWalletIsInstalled(Boolean(isWalletInstalled));
+    if (!isWalletInstalled && availablePlugins) {
+      const walletPlugin = availablePlugins.find(
+        (plugin: PluginStoreItemRequest) => plugin.name === 'MassaWallet',
+      );
+      if (walletPlugin) {
+        setUrlPlugin(walletPlugin.file.url);
+      }
+    }
+  }, [plugins, availablePlugins]);
+
+  useEffect(() => {
+    setDashboardKey((prevKey) => prevKey + 1);
+  }, [pluginWalletIsInstalled]);
+
+  function handleInstallPlugin() {
+    try {
+      mutate({});
+      setPluginWalletIsInstalled(true);
+    } catch (error) {
+      console.error('Error installing plugin:', error);
+    }
+  }
+
   return (
-    <DashboardStation
-      imagesDark={[]}
-      imagesLight={[]}
-      components={[
-        <PluginWallet
-          isActive={true}
-          title={'Massawallet'}
-          iconActive={null}
-          iconInactive={null}
-          onClickActive={() => console.log('install')}
-          onClickInactive={() => console.log('launch')}
-        />,
-      ]}
-    />
+    <>
+      <div className="pt-32 pb-16 text-f-primary mas-banner">
+        Decentralization made easy
+      </div>
+      <div className="flex space-x-8 pb-10">
+        <Button
+          preIcon={<FiGlobe />}
+          customClass="w-96"
+          onClick={() => navigate('/web/search')}
+        >
+          <div className="flex items-center mas-buttons">
+            Search Massa ecosystem
+          </div>
+        </Button>
+        <Button
+          variant="secondary"
+          preIcon={<FiCodepen />}
+          customClass="w-96"
+          onClick={() => navigate('/web/store')}
+        >
+          <div className="flex items-center mas-buttons">
+            Explore module store
+          </div>
+        </Button>
+      </div>
+      <DashboardStation
+        key={dashboardKey}
+        imagesDark={[
+          <Image1Dark key="1" />,
+          <Image2Dark key="2" />,
+          <Image3Dark key="3" />,
+          <Image4Dark key="4" />,
+          <Image5Dark key="5" />,
+        ]}
+        imagesLight={[
+          <Image1Light key="1" />,
+          <Image2Light key="2" />,
+          <Image3Light key="3" />,
+          <Image4Light key="4" />,
+          <Image5Light key="5" />,
+        ]}
+        components={[
+          <PluginWallet
+            key="wallet"
+            isActive={pluginWalletIsInstalled}
+            title="Massawallet"
+            iconActive={<WalletActive />}
+            iconInactive={<WalletInactive />}
+            onClickActive={() =>
+              navigate('/plugin/massalabs/wallet/web-app/index')
```

Comment:
Not your fault but I guess you missed the `routeFor` wrapping. We need that for production links.

```suggestion
              navigate(routeFor('/plugin/massalabs/wallet/web-app/index'))
```

don't forget to import. It's under /utils

`import { routeFor } from '../../../utils';`

Reply:
maybe not here, let's try

Reply:
done

Reply:
I see something I don't understand:

click to navigate to:
```
navigate(routeFor('/web/search'))}
```
I'm in:
![image](https://github.com/massalabs/station/assets/91454320/10662896-a76b-41ca-9ea2-084112ae2943)


click to navigate to:
```
navigate(routeFor('/plugin/massalabs/wallet/web-app/index'))
```
I'm in:
![image](https://github.com/massalabs/station/assets/91454320/4183b22c-b38c-420b-98bd-e634841ed98f)

Reply:
> maybe not here, let's try

Ahh ok! Didn't know! My bad then!

Let's remove it!

Reply:
I remove it, done